src/controllers/competition.controller.ts

```diff
@@ -340,6 +374,39 @@ export class CompetitionController {
    */
   static async getRules(req: Request, res: Response, next: NextFunction) {
     try {
+      // Check if the team is authenticated
+      // @ts-ignore - teamId is added to the request by team-auth middleware
```

Comment:
I don't love this, can we find a way to include it as an optional using an interface extension?

Reply:
agreed - [b059ac2](https://github.com/recallnet/trading-simulator/pull/10/commits/b059ac2931c21b9d7e0b2a6c09db8b59482341c1)